### ISSUE TITLE
sockets dont receive events

### ISSUE THREAD

**Issue by charlitos85** [balderdashy/sails#2467]
I have a custom controller and i cant receive the event of update, create, etc in web socket.
when i dont replace the blueprint action everything all right. whats wrong???

module.exports = {
	find: function(req, res){
		Group.find().exec(function(err, groups){
			if(err)
				res.send(500,err);
			else{
				if(req.isSocket)					
					Group.subscribe(req.socket, groups, ['create','destroy']); 				
				res.send(200, groups);
			}
		})
	},
	create: function(req, res){		
		var data = req.body;

		Group.create(data, function (err, group){
			if(err)
				res.send(500, err);
			else	{			
				Group.publishCreate(group, req.socket);
				res.send(200, group);
                       }
		})
	}
};

**Comment by CWyrtzen** [comment]
Is this still an issue?

**Comment by sailsbot** [comment]
Thanks for posting, @charlitos85.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
